mlir/lib/Dialect/MIGraphX/Transforms/RealizeInt4.cpp

```diff
@@ -0,0 +1,200 @@
+//===- RealizeInt4.cpp ------------------------------------===//
+//
+// Part of the rocMLIR Project, under the Apache License v2.0 with LLVM
+// Exceptions. See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// Copyright (c) 2024 Advanced Micro Devices
+//
+//===----------------------------------------------------------------------===//
```

Comment:
Please add more docs what is pass is supposed to be doing here.

Reply:
It's in the tablegen

Reply:
oh ok.